resources/ApprovePage.js

```diff
@@ -31,7 +17,7 @@ const handleApprovalResponse = ( approve, data ) => {
 	if ( approve ) {
 		const { approver, approvalTimestamp } = data;
 		$( '.approver-name' ).text( approver );
-		convertTimestamps( approvalTimestamp );
+		$( '.convert-timestamp' ).text( approvalTimestamp );
```

Comment:
This looks like a behavior change. You stopped using moment and switched `approvalTimestamp` from int to formatted string in the backend. So now it will always show some formatted time instead of "some seconds ago"?

Reply:
I see this got replaced by `getHumanTimestamp` on the backend